src/reflect/scala/reflect/internal/util/Position.scala

```diff
@@ -94,6 +94,8 @@ sealed abstract class UndefinedPosition extends Position {
   override def start           = fail("start")
   override def point           = fail("point")
   override def end             = fail("end")
+
+  override def samePointAs(that: Position) = !that.isDefined
```

Comment:
Coming back to this, I waffle slightly on this definition. I'm willing to be convinced that the RHS should be `false`.

Reply:
`false` sounds more reasonable.

Reply:
You've convinced me.